### ISSUE TITLE
Afibrinogenemia

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7535]
**Preferred term label:**
Afibrinogenemia

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
202400 
PMID: 7363509  10/10
PMID: 10891444  13/13

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
